dev-qt/qt5/qt5-5.6.1.recipe

```diff
@@ -9,7 +9,7 @@ LICENSE="GNU LGPL v2"
 
 SOURCE_URI="https://download.qt.io/official_releases/qt/5.6/$portVersion/single/qt-everywhere-opensource-src-$portVersion.tar.xz"
```

Comment:
I don't get where this is going again. Xz is a significant download and space gain for a piece like Qt. And Qt is out of question on x86_gcc2, so this point is moot.